lib/spark_api/models/listing.rb

```diff
@@ -235,10 +235,25 @@ def respond_to?(method_symbol, include_all=false)
         end
       end
 
-      def batch_photo_delete(photoIds, args={})
+      def batch_photo_delete!(photoIds, args={})
         connection.delete "#{self.class.path}/#{self.Id}/photos/#{photoIds}", args
       end
 
+      def batch_photo_delete(photoIds, args={})
```

Comment:
I think `delete_photos` is more consistent with existing naming (e.g. `reorder_photos`).